### DISCUSSION TITLE
Confirmation number for Incomplete BWMR?

### DISCUSSION THREAD

**Discussion by tamihuber** [paulwinterbauer/NBIC#152] [Q&A] [answered]
Wasn't sure if this is supposed to happen. I thought if it gets an incomplete response, it doesn't get assigned a confirmation #. 

![image](https://user-images.githubusercontent.com/100441006/172377257-16659a12-c9e8-42ed-9f7e-7be14e2a9bd1.png)

**Comment by karlsi** [comment]
Correct.  Incomplete reports do not have include the confirmation number in the auto-reply email.   That is supposed to encourage them to submit a complete report.

**Comment by tamihuber** [comment]
But this one shows a confirmation number. Highlighted in the report pic above.

**Comment by karlsi** [comment, marked as answer]
The confirmation number is stored in the database, but it is not included in the auto-reply.

**Comment by tamihuber** [comment]
Ah, gotcha. I never noticed that before! Thanks!
